utilities/boostHistHelpers.py

```diff
@@ -13,9 +13,27 @@ def valsAndVariances(h1, h2):
 def broadcastSystHist(h1, h2):
     if h1.ndim > h2.ndim or h1.shape == h2.shape:
         return h1
-    
+
+    a1 = h1.view(flow=True)
+    a2 = h2.view(flow=True)
+
+    # the additional axes have to be trailing, if they ar not, they must be moved
```

Comment:
This code is probably faster but it might be easier to just do two project calls

Reply:
Yes, I think now only views are used while projecting is much more expensive. Given that broadcasting function is used a lot I think it's important that this is fast.

Reply:
Yeah fair. It seems like you can pass a sequence to np.moveaxis, so you could do all the axis moves at once which avoids the loop and is a little cleaner

Reply:
Done